src/meta/expr.lean

```diff
@@ -128,6 +128,10 @@ end level
 namespace expr
 open tactic
 
+/-- Apply `e` to all elements of `es` -/
+meta def apps (e : expr) (es : list expr) : expr :=
```

Comment:
this is a repeat of `expr.mk_app`

Reply:
Oh thanks, I should have looked more carefully!